src/content/docs/en/basics/layouts.mdx

```diff
@@ -234,6 +234,12 @@ const { title, fancyJsHelper } = Astro.props;
 
 <ReadMore>Learn more about Astro’s Markdown and MDX support in our [Markdown guide](/en/guides/markdown-content/).</ReadMore>
 
+:::note
```

Comment:
I think on the Layouts page, the better way to incorporate this content is to work it into the description of the layout throughout the section so that it's more of, "your layout component does this" and update all the code examples to always show that meta tag. For example, stuff like (and, not actually in bold!):

> Markdown Layouts
> Page layouts are especially useful for individual Markdown pages which otherwise would not have any page formatting.
>
>Astro provides a special layout frontmatter property intended for [individual .md files located within src/pages/ using file-based routing](/en/guides/markdown-content/#individual-markdown-pages) to specify which `.astro` component to use as the page layout. **This component allows you to provide `<head>` content like meta tags (e.g. `<meta charset="utf-8">`) and styles for the Markdown page.** By default, this specified component can automatically access data from the Markdown file.
>
>This is not recognized as a special property when using [content collections](https://5-0-0-beta.docs.astro.build/en/guides/content-collections/) to query and render your content.

If you're OK with that idea, then I can go through and add stuff like this sprinkled in the appropriate places! Then, I might add something to the Markdown Pages section itself re: what happens if you **don't** have a layout. But I think on the layout page here, we can simply show by example. (Also quick reminder that docs itself wouldn't describe what Astro "doesn't do any more", but only how it works now!)

Reply:
That sounds good to me! Feel free to update it to add that information.

I added a simple note to not have a drastic change, but a complete face-lift is always better to make it flow nicely 😄

Reply:
great! Leave this one with me and I'll add a couple of things!